tm2/pkg/sdk/baseapp_test.go

```diff
@@ -753,7 +786,7 @@ func TestSimulateTx(t *testing.T) {
 		require.True(t, queryResult.IsOK(), queryResult.Log)
 
 		var res Result
-		amino.MustUnmarshal(queryResult.Value, &res)
+		json.Unmarshal(queryResult.ResponseBase.Data, &res)
```

Comment:
Why switch from `amino`?

Reply:
Updated in:
https://github.com/gnolang/gno/pull/3207/commits/ef4ca47d4db3f5ecc1638c88b71f39337a228ea3